rpc_jobs/snapshot.yml

```diff
@@ -0,0 +1,95 @@
+- job:
+    name: RE35-Snapshot-Test
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      # Default params are provided by macro, add any extra params, or
+      # params you want to override the defaults for.
+      - bool:
+          name: CREATE_RPCO_SNAPSHOT
+          default: true
+          description: Run RPCO Build and save as snapshot.
+      - bool:
+          name: BUILD_FROM_SNAPSHOT
+          description: |
+            Build instance from RPCO snapshot and run tempest.
+            Note that the IMAGE parameter should contain the
+            name of the snapshot to build from. The IMAGE
+            paramter is not passed through to the RPC AIO job,
+            which will use its own default image.
+          default: true
+      - instance_params:
+          IMAGE: "rpc_r15.0.0"
```

Comment:
r15.0.0 ?

Reply:
One thing I've not quite figured out yet is how we handle releases like newton that can be built on either Trusty or Xenial.

Reply:
This snapshot job is just for testing. Production images will be created by push triggered post merge jobs.

Reply:
Does this image need to be updated to reflect the distro that now gets added to the image name?